ports/javascript/Makefile

```
@@ -20,13 +20,23 @@ BOTH_CFLAGS := -s WASM=0
 
 CPP = clang -E
 
+#for lvgl
+CPPFLAGS += -I$(EM_CACHE)/asmjs/ports-builds/sdl2/include
+
+ifdef WASM_FILE_API
+	CPPFLAGS += -DWASM_FILE_API=1 
+endif
+
 ifdef EMSCRIPTEN
-    CPP += -isystem $(EMSCRIPTEN)/system/include/libc -isystem $(EMSCRIPTEN)/system/include/libcxx -isystem $(EM_CACHE)/asmjs/ports-builds/sdl2/include -isystem $(EMSCRIPTEN)/system/include
+    CPP += --sysroot $(EMSCRIPTEN)/system
```

The `sysroot` line makes Emscripten fail to find SDL2 includes. My original method seems to work fine.

```suggestion
-    CPP += --sysroot $(EMSCRIPTEN)/system
+   CPP += -isystem $(EMSCRIPTEN)/system/include
```

---

fixed, i had forgot to add CPP += $(CPPFLAGS)